Model/AbstractApi.php

```diff
@@ -160,11 +194,16 @@ public function sendApiRequest($path, array $data, $method = "post", $timeout =
      * @method oauthAuthentication
      * @param  int|null $scopeId
      * @param  string|null $scope
+     * @param bool $forceFlag
```

Comment:
please change the forceFlag name to: forceCreateTokenFlag

Reply:
Sure, will change it as per mentioned